### PR TITLE
Skill validation

### PR DESCRIPTION
**Pull request by gabimartini**
trello issue: https://trello.com/c/gc6ahud
Screenshot 2022-05-18 at 10 41 54
Screenshot 2022-05-18 at 10 44 21

### CODE REVIEW

**Review by em-cd — COMMENTED** [review]
I'm wondering whether it would make sense to split the errors up a bit more so that we can make the error messages easier to understand.

So for example if someone checks 'food hygiene training' and doesn't add the date, we could show a message like: 'Please add the date this person achieved food hygiene training'

And for the opposite situation: 'Please tick food hygiene training field or remove the date achieved'

Not sure if that makes sense, what do you think @gabimartini?

File: spec/features/managing_employees_spec.rb [hunk 1]
```diff
@@ -104,12 +104,24 @@
           fill_in :employee_date_of_birth, with: dob
           page.select job_title, from: :employee_job_title
           check :employee_currently_employed
-
+          check :employee_has_food_hygiene
+          check :employee_has_dbs_check
+          check :employee_has_first_aid_training
+          check :employee_has_senco_training
+          check :employee_has_senco_early_years
+          check :employee_has_safeguarding
```
Comment (em-cd) [inline spec/features/managing_employees_spec.rb:112]: Love these tests! 😍

**Comment by gabimartini** [comment]
@em-cd what do you think, now?

**Review by em-cd — APPROVED** [review]
Looks good! Thanks @gabimartini 😄

**Pull request merged**